src/api-types.ts

```diff
@@ -56,27 +60,27 @@ export interface AppClientNewFormBody {
   /**
    * Can the client can use the 'client_credentials' flow
    */
-  allowClientCredentials?: "on";
+  allowClientCredentials?: string;
   /**
    * Can the client can use the 'authorization_code' flow
    */
-  allowAuthorizationCode?: "on";
+  allowAuthorizationCode?: string;
   /**
    * Can the client can use the OAuth 2.0 for First Party Applications flow
    */
-  allowAuthorizationChallenge?: "on";
+  allowAuthorizationChallenge?: string;
   /**
    * Can the client can use the deprecated 'implicit' flow
    */
-  allowImplicit?: "on";
+  allowImplicit?: string;
   /**
    * Is the client allowed to refresh tokens
    */
-  allowRefreshToken?: "on";
+  allowRefreshToken?: string;
   /**
    * Can the client can use the 'password' flow flow
    */
-  allowPassword?: "on";
+  allowPassword?: string;
```

Comment:
Ignore all these. looks like `api-types` wasn't fully updated, so this automatically got updated too. But it's not relevant for your PR

Reply:
This was the same type changes it gave me when I ran them

Reply:
I see we have tech debt! !!!! 😱

Reply:
lol yes. plenty haha